content/S2016/S16-event-teaparty.md

```diff
@@ -0,0 +1,25 @@
+title: Tea Party and Volunteer Meeting
+Date: 2016-4-27
+Category: Events
+Tags: social, volunteer, big csters
+Slug: tea-party-s16
+Author: Evy Kassirer
+Summary: Join us for tea and for information on volunteering with us!
+
+Welcome back! You're cordially invited to our Tea Party and Volunteer meeting.
+Come out to socialize with some WiCS members, meet the new committee, and learn
+more about our volunteer opportunities! The volunteer meeting will run from
+5:00 - 5:30, and the Tea Party will go from 5:30 to 7:00. If you just want to
+come for the tea party, feel free to come by at 5:30. Please try to bring your
+own mug if possible.
+
+Note: Although our volunteer meeting is open to everyone, our Tea Party is
+a Big CSters event is and is thus [women-only](http://wics.uwaterloo.ca/faq).
+
+## Event Details ##
+
++ **Who:** Everyone for volunteer meeting,
+           [women-only](http://wics.uwaterloo.ca/faq) for tea party
```

Comment:
Sorry to make a nit again, but could we link "women and non-binary individuals" instead of "women-only"? This sentence consists of $noun and $adjective. I will Heroku-deploy in the interim to check the date thing above.